.idea/jarRepositories.xml

```diff
@@ -1,6 +1,11 @@
 <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I prefer to not have .idea config files in the repository. 
I have troubles building the project in these cases quite often.

Reply:
Idea file is removed from the gothub repository.

Reply:
Now the ./idea file is removed